howto/howto-get-set-remove-contact-raw-contact-photo.md

```diff
@@ -161,21 +161,6 @@ val insertResult = Contacts(context)
 
 > For more info about insert, read [How do I create/insert contacts?](/howto/howto-insert-contacts.md)
 
-It is true that the API should not allow you to call `setPhoto` on a Contact/RawContact that has not
-yet been inserted. It is currently misleading. This will be fixed in issue #117. As of right now, 
-the following documentation is included in `contacts.core.util.ContactPhoto.kt` and 
-`contacts.core.util.RawContactPhoto.kt` functions.
-
-```kotlin
-/*
- * ## For existing (inserted) entities only
```

Comment:
Documentation is no longer needed for existing entity functions because the compiler checks that now.